d2l-icon-button.html

```diff
@@ -1,23 +1,20 @@
 <link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../d2l-offscreen/d2l-offscreen-shared-styles.html">
 <link rel="import" href="d2l-icon.html">
 <link rel="import" href="d2l-icon-button-shared-styles.html">
 <link rel="import" href="d2l-icon-button-behavior.html">
 
 <dom-module id="d2l-icon-button">
 	<template strip-whitespace>
-		<style include="d2l-icon-button-shared-styles d2l-offscreen-shared-styles">
-			:host > span {
-				@apply(--d2l-offscreen);
-			}
+		<style include="d2l-icon-button-shared-styles">
 		</style>
-		<d2l-icon icon="[[icon]]"></d2l-icon>
-		<span><content></content></span>
+		<button class="d2l-icon-button" disabled={{disabled}}>
```

Comment:
Similar to my comment about `icon-link` -- all the attributes the consumer put on the `<button>` element before are now stuck on the `<d2l-icon-button>` and aren't getting transferred.